Source/BSN.Commons.S3.Abstraction/BSN.Commons.S3.Abstraction.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
I think maybe is better to merge this project into BSN.Commons

Reply:
what is your opinion?

Reply:
I moved everything into BSN.Commons.

Looks much more better.